packages/xo-server/src/_handleBackupLog.mjs

```diff
@@ -6,8 +33,44 @@ function forwardResult(log) {
 }
 
 // it records logs generated by `@xen-orchestra/backups/Task#run`
-export const handleBackupLog = (log, { logger, localTaskIds, rootTaskId, runJobId = rootTaskId, handleRootTaskId }) => {
-  const { event, message, taskId } = log
+export const handleBackupLog = (
+  app,
+  jobName = undefined,
+  log,
+  { logger, localTaskIds, rootTaskId, runJobId = rootTaskId, handleRootTaskId }
+) => {
+  const { event, message, parentId, taskId } = log
+
+  if (jobName) {
```

Comment:
Explicit tests, only pass true booleans to `if`s.

Reply:
I thought that in JS ```undefined``` or ```null``` are falsy values so ```jobName``` would be a truthly value in any other case, wouldn't it?

Reply:
That's true, but in the project we prefer to use explicit comparison because it makes the code easier to maintain.